File: src/main/java/net/hectus/neobb/turn/person_game/block/PTSeaLantern.java
package net.hectus.neobb.turn.person_game.block;

import net.hectus.neobb.player.NeoPlayer;
import net.hectus.neobb.turn.default_game.block.BlockTurn;
import net.hectus.neobb.turn.person_game.categorization.UtilityCategory;
import org.bukkit.block.Block;

public class PTSeaLantern extends BlockTurn implements UtilityCategory {
    public PTSeaLantern(NeoPlayer player) { super(player); }
    public PTSeaLantern(Block data, NeoPlayer player) { super(data, player); }

    @Override
    public void apply() {
        super.apply();
        player.addArmor(1);
    }
}

File: src/main/kotlin/net/hectus/neobb/turn/person_game/block/PTSeaLantern.kt
package net.hectus.neobb.turn.person_game.block

import com.marcpg.libpg.storing.Cord
import net.hectus.neobb.player.NeoPlayer
import net.hectus.neobb.turn.default_game.block.BlockTurn
import net.hectus.neobb.turn.person_game.categorization.UtilityCategory
import org.bukkit.block.Block

class PTSeaLantern(data: Block?, cord: Cord?, player: NeoPlayer?) : BlockTurn(data, cord, player), UtilityCategory {
    override fun apply() {
        player!!.addArmor(1.0)
    }
}